src/__tests__/globalizeSelectors-spec.js

```diff
@@ -8,25 +8,34 @@ describe('globalizeSelectors', () => {
       numbers: [2, 3, 4]
     }
   }
-
   const localState = prop('section')
-
-  const selectors = {
-    numbers: prop('numbers'),
-    numberAt: (index, state) => state.numbers[index]
-  }
-
-  const globalized = globalizeSelectors(localState, selectors)
+  const numbers = prop('numbers')
+  const numberAt = (index, state) => state.numbers[index]
+  const globalized = globalizeSelectors(localState, {
+    numbers,
+    numberAt,
+    curriedNumberAt: curry(numberAt)
+  })
 
   context('with a single-argument selector', () => {
-    test('allows the selector to work from the global state', () => {
+    it('allows the selector to work from the global state', () => {
       expect(globalized.numbers(globalState)).toEqual([2, 3, 4])
     })
   })
 
   context('with a multi-argument selector', () => {
-    test('allows the selector to work from the global state', () => {
-      expect(globalized.numberAt(1, globalState)).toEqual(3)
+    it('allows the selector to work from the global state', () => {
```

Comment:
I like the idea of using `it` since it reads more similarly to the other testing frameworks we use. Is `test` is more idiomatic for Jest? If so, maybe we should revert back to that.